### PR TITLE
Frontend

### PR DESCRIPTION
**Pull request by deamme**

### CODE REVIEW

**Comment by osarrouy** [comment]
Hey !

Just had a quick look. Its looks really great! A a few comments:

- [ ] It could be cool to re-factor the logic to use the new `api-react` library documented [here](https://hack.aragon.org/docs/aragonjs-ref-react).
- [ ] It would make sense too to make sure the UX is responsive from scratch.
- [ ] Regarding the `new order` side panel we should introduce two tabs inside: one for the sell orders and one for the buy orders.
- [ ] When one hover the bar charts would it be possible to display the date / time and price?
- [ ] A small tweak but we're gonna use DAI instead of ETH as the main collateral.
- [ ] The bonding curve / history chart menu is reversed :)

Otherwise the rest looks really nice!

I'm gonna try to provide you some mocks data tomorrow so that you guys can have a clearer idea of how these data will look like!

Congrats.

File: apps/apiary/app/src/App.js [hunk 1]
```diff
@@ -1,144 +1,107 @@
...
 import Orders from './screens/Orders'
 import Overview from './screens/Overview'
 
 const tabs = ['Overview', 'Buys / Sells', 'Settings']
 
-class App extends React.Component {
-  static defaultProps = {
+const App = () => {
+  const [state, setState] = useState({
     repos: [],
```
[the remark this answers is not in the document]
Comment (deamme) [reply apps/apiary/app/src/App.js]: It is, I'll clean it up!

**Comment by osarrouy** [comment]
Hey @deamme,

Nice job. Another batch of comments:

## Orders tab
- [ ] The `Rate` column should be renamed `Price`
- [ ] We should add a `State` column which would display states of orders as `Badges` being either `Pendind`, `Cleared` or 'Returned`. This is due to how the bonding curve batches orders and thus works: you first open a order, then wait for the batch to be cleared [or clear it yourself] and then claim it to actually get your tokens.

## Overview tab
- [ ] I feel like the date selector [and the 1 day, 1 week, etc. selector] has no effect for now on what is actually displayed ...
- [ ] When the Curve view is displayed it would be cool to a small point on the curve to figure out where we are right now on the price curve.

## New 'My orders tab'
- [ ] I think we should add a 'My orders' tab which would basically be the same than the 'Buy / sell'  tab excepts that it would only display the current account orders. Also in the action menu of these orders rows we should add a 'clear' or 'claim' action depending on what is the state of the order.

**Comment by cryptodani** [comment]
Hi @deamme,
It's starting to look fantastic! A few comments from a fan if they fit in the timeline:

[ ] We should probably add decimal display to BC token supply (now displays "5600")
[ ] Buying order" and "Selling Order" would sound better as "Buy order" and "sell order"or simply "Buy" and "Sell"
[ ] When clicking on address badge it should display the new ui component with custom badge  (identity tagging)
[ ] "Returned" should be changed for clarity to "Cancelled" if this is what is meant
[ ] In order tabs Amount of collateral should be accompanied by Amount of DAO/BC token bought/sold

Bonuses:
-Switching fiat currency conversion from USD to others such as EUR/YEN/CNY if available from oracle (probably for a later release)
-Collateralization Ratio/CW display (also pushable to later release to be able to distinguish between BCs volatility)

**Comment by deamme** [comment]
Closing this one since we want to merge dev into master instead

**Pull request closed without merging**